src/Classes/Particle/Bubbler.ts

```diff
@@ -59,13 +59,7 @@ export class Bubbler {
         if (bubbleParam !== particlesParam) {
             if (!container.bubble.durationEnd) {
                 if (distMouse <= bubbleDistance) {
-                    let obj;
-
-                    if (pObjBubble) {
-                        obj = pObjBubble;
-                    } else {
-                        obj = pObj;
-                    }
+                    let obj = pObjBubble ?? pObj;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'obj' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5238794)